test/e2e/tests/test_tenant.py

```
@@ -0,0 +1,175 @@
+TENANT_NAMESPACE = os.environ.get("DEPLOYMENT_NAMESPACE", "opendatahub")
+TENANT_CRD = "tenants.maas.opendatahub.io"
+
+_KIND_PLURAL = {
+    "maasmodelref": "maasmodelrefs",
+    "maasauthpolicy": "maasauthpolicies",
+    "maassubscription": "maassubscriptions",
+}
+
+
+def _oc_json(args):
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)